service_broker/service_broker_app.rb

```diff
@@ -119,7 +119,8 @@ def self.app_settings
   end
 
   def github_service
-    github_credentials = self.class.app_settings.fetch("github")
-    GithubServiceHelper.new(github_credentials.fetch("username"), github_credentials.fetch("access_token"))
+    gh_user = ENV["GITHUB_USER"]
+    gh_token = ENV["GITHUB_TOKEN"]
+    GithubServiceHelper.new(gh_user, gh_token)
```

Comment:
fixed in https://github.com/deis/cloudfoundry-github-service-broker-ruby/pull/5/commits/b81442747a2e6bccc3e1a7e35ce8f99c7a04fb08 - thanks!